kenwood/tmd710.c

```diff
@@ -35,6 +48,7 @@
 #include "tones.h"
 #include "num_stdio.h"
 #include "misc.h"
+#include "../include/hamlib/rig.h"
```

Comment:
Please replace this with `#include <hamlib/rig.h>` as this is an installed header and should be included in this manner per GNU documentation.

Reply:
Sure. That was inserted by the IDE and I didn't catch it :)

Reply:
Actually, the file was already included, so just removing the line.

Reply:
I see that now and even there it is included incorrectly.  Sigh.  I'll get that corrected "one of these days".  One thing to be aware of is that relative include paths like that will probably break VPATH builds and would be caught by running `make distcheck` which is the main way that I check for regressions.